jwst/flatfield/flat_field.py

```diff
@@ -1163,34 +1153,28 @@ def read_flat_table(flat_model, exposure_type, slit_name=None, quadrant=None):
     flat_model : NIRSpec flat-field object
         This contains the flat field table from which we will read the
         "fast" variation flat-field data.
-
     exposure_type : str
         The exposure type refers to fixed_slit, IFU, or using the
         micro-shutter array.  In this function we just need to check for
         fixed-slit types.
-
-    slit_name : str
+    slit_name : str or None, optional
         The name of the slit.  This is only needed for fixed-slit data, in
         which case it is used for selecting the relevant row of the table.
-
-    quadrant : int (0, 1, 2, or 3)
+    quadrant : {0, 1, 2, 3} or None, optional
```

Comment:
Is this the standard way to indicate a choice of integers?  To me on first reading it looked like this should accept the set `{0, 1, 2, 3}`

Reply:
perfect thanks, yeah I think either of the two ways you were debating would make sense